code/modules/RCD/schematics/pipe.dm

```diff
@@ -152,8 +152,8 @@
 
 	to_chat(user, "Building gas sensor...")
 	playsound(get_turf(master), 'sound/machines/click.ogg', 50, 1)
-	if(!do_after(user, A, 20))
-		return 1
+//	if(!do_after(user, A, 1))
```

Comment:
Does this mean I need to close the PR and re do it? Also I made the branch off of bleeding edge